templates/docs/patterns/hero/index.md

```diff
@@ -101,5 +195,19 @@ View example of the hero pattern in fallback configuration
 
 ## Import
 
+### Jinja Macro
+
+To import the Hero Jinja macro, copy the following import statement into your Jinja template.
+
+```jinja
+{% raw -%}
```

Comment:
Why the `-%}` is only in the end? Would it make sense to use `{%- raw -%}` and `{%- endraw -%}` for consistency, or does it affect white spaces in some way?

Reply:
```jinja
{%- raw -%}
{% from "_macros/vf_hero.jinja" import vf_hero %}
{%- endraw -%}
```
Strips necessary markdown spacing and shows the markdown code block name
```
jinja{% from "_macros/vf_hero.jinja" import vf_hero %}
```

Looks like the best we can do there is this:

```jinja
{% raw -%}
{% from "_macros/vf_hero.jinja" import vf_hero %}
{%- endraw -%}
```
<img width="615" alt="Screenshot 2024-08-30 at 12 08 37 PM" src="https://github.com/user-attachments/assets/3d6afa5a-b04a-445b-8d51-d064c6bf56e5">

Reply:
Revised as seen above